assets/js/amp-invalid-url-post-edit-screen.js

```diff
@@ -391,5 +393,17 @@ const ampInvalidUrlPostEditScreen = ( function() { // eslint-disable-line no-unu
 		}
 	};
 
+	/**
+	 * Adds the AMP icon to the page heading if AMP is enabled on this URL.
+	 */
+	component.showAMPIconIfEnabled = function() {
+		const heading = document.getElementsByClassName( 'wp-heading-inline' );
```

Comment:
This can use `querySelector` instead of `getElementsByClassName`. So:

```js
const heading = document.querySelector( '.wp-heading-inline' );
```

Then below just check to see if `heading` is truthy as opposed to `heading[0]`